.buildkite/dagster-buildkite/dagster_buildkite/python_version.py

```diff
@@ -21,6 +21,11 @@ def get_all(cls) -> List["AvailablePythonVersion"]:
     def get_default(cls) -> "AvailablePythonVersion":
         return cls["V3_11"]
 
+    # Useful for excluding everything but the default
```

Comment:
nit: Should the comment be "excluding the default"?

Reply:
👍 tweaked comment for clarity